changelog/inclusive-incontracts.dd

```diff
@@ -0,0 +1,25 @@
+Add `-preview=inclusivein`: `in` contracts must be an explicit superset of the parent `in` contracts.
```

Comment:
Given that we have `-preview=in` could you maybe change this to `-preview=inclusiveincontracts` to avoid any confusion.

Reply:
That's so long that `-preview=h` falls over.
```
  =inclusiveincontracts'in' contracts of overridden methods must be a superset of parent contract
```
Do it anyway?

Reply:
added an extra space there